modules/luci-mod-status/htdocs/luci-static/resources/view/status/index.js

```diff
@@ -1,13 +1,15 @@
-function progressbar(q, v, m)
+function progressbar(query, value, max, byte)
 {
-	var pg = document.querySelector(q),
-	    vn = parseInt(v) || 0,
-	    mn = parseInt(m) || 100,
+	var pg = document.querySelector(query),
+	    vn = parseInt(value) || 0,
+	    mn = parseInt(max) || 100,
+	    fv = byte ? String.format('%.2mB ', value) : value;
+	    fm = byte ? String.format('%.2mB ', max) : max;
```

Comment:
It should be comma as the line end, not semicolon.

Reply:
oops, sorry for that one. Fixed it!